one-on-ones.md

```diff
@@ -0,0 +1,25 @@
+# Project 2 One-on-One Objectives
+
+## Friday (15 minutes)
+- Communicate and lingering concerns and/or confusion about Unit 2 topics.
+- Identify at least one topic to review or research over the weekend in preparation for Project 2
+- Discuss possible project ideas and identify strong candidates
+
+### Homework:
+
+- Create a new repository on GitHub for your project.
+  - Create `readme.md` file in the root of your repo with a a 1-2 sentence summary of your project idea.
+  - Create a `planning` directory in the root of your repo, containing...
+    - A `wireframes` directory, inside of which are *at least* 2 photos or drawings of wireframes
+    - A `user_stories.md` file, containing at least 7 user stories, *or* a link to Trello with at least 7 user stories.
+- Create an issue on *this* repository with a link the repository you created above by **9:00am on Monday, November 16<sup>th</sup>**.
+
```

Comment:
We should probably cover how to do `rails new my_app .` to create a rails app in the current folder, rather than nested.